gtsfm/densify/patchmatchnet_data.py

```diff
@@ -0,0 +1,273 @@
+"""Interface class from GtsfmData to PatchmatchNetData
+
+Authors: Ren Liu
+"""
+from collections import defaultdict
+from typing import Any, Dict, List, Tuple
+
+import cv2
+import numpy as np
+from torch.utils.data import Dataset
+
+from gtsfm.common.gtsfm_data import GtsfmData
+from gtsfm.common.image import Image
+import gtsfm.densify.mvs_utils as mvs_utils
```

Comment:
`import *` lines should be placed above `from * import * `lines

Reply:
```python
from collections import defaultdict
from typing import Any, Dict, List, Tuple

import cv2
import numpy as np
from torch.utils.data import Dataset

from gtsfm.common.gtsfm_data import GtsfmData

import gtsfm.densify.mvs_utils as mvs_utils
from gtsfm.common.image import Image
```
we can add to contributing guidelines/python

Reply:
thanks, resolved